packages/content/src/Application/PropertyResolver/Resolver/TeaserSelectionPropertyResolver.php

```diff
@@ -32,7 +32,7 @@ public function resolve(mixed $data, string $locale, array $params = []): Conten
             || !\array_key_exists('items', $data)
             || !\is_array($data['items'])
         ) {
-            return ContentView::create($data, $returnedParams);
```

Comment:
Can you also add a small test for this in the TeaserSelectionPropertyResolverTest ?

Reply:
@Prokyonn good catch 👍  added